Integrations/test_python/pyMethods.py

```diff
@@ -0,0 +1,16 @@
+class pyMethod:
```

Comment:
What is this used for?

Reply:
Shouldn't be in there, I think I created it to have some python functions to call in testing, but never used it. It should not have been committed at all, but I don't know if I can remove it from the commit. I'll see.